pkg/filter/util/cloudcost.go

```diff
@@ -11,13 +11,13 @@ import (
 )
 
 type CloudCostFilter struct {
-	AccountIDs       []string `json:"accountIDs"`
-	Categories       []string `json:"categories"`
-	InvoiceEntityIDs []string `json:"invoiceEntityIDs"`
-	Labels           []string `json:"labels"`
-	Providers        []string `json:"providers"`
-	ProviderIDs      []string `json:"providerIDs"`
-	Services         []string `json:"services"`
+	AccountIDs       []string `json:"accountIDs,omitempty"`
+	Categories       []string `json:"categories,omitempty"`
+	InvoiceEntityIDs []string `json:"invoiceEntityIDs,omitempty"`
+	Labels           []string `json:"labels,omitempty"`
+	Providers        []string `json:"providers,omitempty"`
+	ProviderIDs      []string `json:"providerIDs,omitempty"`
+	Services         []string `json:"services,omitempty"`
```

Comment:
Is this struct used for any responses other than the `groupReport` configuration (maybe cloud cost reports)?

The change to `omitempty` could affect other consumers if so. If this is localized entirely to `/getGroupReports`, then nevermind!

Reply:
there are no external uses outside of group reports, so there is no impact. It is used internally inside the application, but since it is not treated as JSON, will have no impact.